public/js/NeighbourImages.js

```diff
@@ -83,48 +102,83 @@ NeighbourImages.prototype.getImages = function(sample_id) {
  */
 NeighbourImages.prototype.setImages = function() {
     var self = this;
-    var $nebula0 = $('#nebula-0');
-    var $body = $('body');
-
-    console.log(self.neighbourImages);
 
     // make all img frames empty
-    $nebula0
+    this.selectors.$nebula0
         .attr('src', '//:0')
         .attr('title', 'not found');
 
-    $('.thumbnail > img')
+    this.selectors.$thumbnailChildImg
         .attr('src', '//:0')
         .attr('title', 'not found');
```

Comment:
Question: when you skip this step, do you end up "stacking" the images on top of one another, hogging resources? Or is this for something else I don't understand?